runners/flink/flink_runner.gradle

```diff
@@ -141,8 +148,10 @@ dependencies {
   implementation library.java.joda_time
   implementation library.java.args4j
 
-  // Flink 1.15 shades all remaining scala dependencies and therefor does not depend on a specific version of Scala anymore
-  if (flink_version.compareTo("1.15") >= 0) {
+  /* Flink 1.15 shades all remaining scala dependencies and therefor does not depend on a specific version of Scala anymore.
+   * Flink 1.16 onwards, flink version is 11600.* and does not depend on a specific version of Scala anymore.
+   */
+  if (flink_version.compareTo("1.15") >= 0 || flink_version.compareTo("11600.3") >= 0) {
```

Comment:
I'm curious about this change: is it 11600.3 >= 1.15? If so, do we need the second part of the condition?

Reply:
@xinyuiscool I feel like post `1.15` flink version, we have a different versioning for flink dependency which is `11600.3` and so on. So, adding an additional condition can better help understand the Flink versioning.
11600.3 >= 1.15 is true and I can make this change. Please suggest.

Reply:
Would it be better to have an indirection say if the `flink_version` is `1.16` then use the corresponding `linkedin_flink_version` which is `11600.3` through a map. That way, when a new version is released in LinkedIn, we just need to change in the map. Also this way even if LinkedIn Flink versioning scheme changes to Semantic versioning with its own `major.minor.patch` version, it won't affect.